backend/resolvers/mutations/show.js

```diff
@@ -9,6 +9,17 @@ export function createShow (_, args, req) {
   return Show.create(args.input)
 }
 
+export function updateShow (_, args, req) {
+  // Only admins can update entries
+  if (req.auth.type !== ADMIN) {
+    throw new UserError('Permission Denied')
+  }
+  return Show.findById(args.id)
+    .then((show) => {
+      return show.update(args.input)
```

Comment:
You should only add update the columns you expect – [`options.fields`](http://docs.sequelizejs.com/class/lib/model.js~Model.html#instance-method-save) is what you want